### DISCUSSION TITLE
is there a C# sdk available ?

### DISCUSSION THREAD

**Discussion by madhugilla** [qdrant/qdrant#2288] [General]
Hello,

we are primarily a MS shop and want to start leveraging Qdrant,  is there a C# sdk available?

Thanks -GM

**Comment by timvisee** [comment]
Qdrant has no official C# client (or SDK) at this time.

Microsoft created [Semantic Kernel](https://github.com/microsoft/semantic-kernel), which does have [support](https://devblogs.microsoft.com/semantic-kernel/the-power-of-persistent-memory-with-semantic-kernel-and-qdrant-vector-database/) for Qdrant through their memory interface.

There also are some community efforts such as https://github.com/russcam/qdrant-dotnet-client
